quiz/settings.py

```diff
@@ -1 +1 @@
-QUIZZ_PATH = "quizzes"
+QUIZ_PATH = "quizzes"
```

Comment:
I think that we should directly append this to the base directory, to avoid it being created anywhere working directory is currently set. In theory, the code working with this variable could do it, but I think that it's better if we do it in one place.
```python
import os
from kelvin.settings import BASE_DIR

QUIZ_PATH = os.path.join(BASE_DIR, "quizzes")
```